nc/source/component/Transform.h

```diff
@@ -25,7 +25,7 @@ namespace nc
             Quaternion GetRotation() const;
             Vector3 GetScale() const;
 
-            DirectX::XMMATRIX GetTransformationMatrix() const;
+            DirectX::FXMMATRIX GetTransformationMatrix() const;
```

Comment:
Why FXMMATRIX?

Reply:
It is a typedef for const & or const depending on platform.